fedn/network/clients/grpc_handler.py

```diff
@@ -191,7 +218,7 @@ def get_model_from_combiner(self, id: str, client_name: str, timeout: int = 20)
         data = BytesIO()
         time_start = time.time()
         request = fedn.ModelRequest(id=id)
-        request.sender.name = client_name
+        request.sender.client_id = client_name
```

Comment:
Parameter `client_name` in `get_model_from_combiner` is the client name - not the client id - as this function is currently used.

Maybe the parameter `client_name` should be replaced by `client_id`?